src/translations/maps/indexMap.js

```diff
@@ -121,6 +122,23 @@ class IndexMap {
     this.runLocalHooks('change');
   }
 
+  /**
+   * Get indexes basing on searched value in the map.
+   *
+   * @private
+   * @param {boolean} value Searched value in the map.
+   * @returns {Array}
+   */
+  getIndexesByValue(value) {
+    return arrayReduce(this.getValues(), (indexesList, valueInMap, index) => {
+      if (valueInMap === value) {
+        indexesList.push(index);
+      }
+
+      return indexesList;
+    }, []);
```

Comment:
`arrayFilter` will not be more convenient helper for this use case? https://github.com/handsontable/handsontable/blob/560eb12677ec1b6b02d86b1f7d9715410f69d903/src/helpers/array.js#L102

```suggestion
    return arrayFilter(this.getValues(), (valueInMap) => {
      return valueInMap === value;
    });
```

Reply:
Nope. We do not filter booleans, we try to get indexes.

Reply:
Right!